OmniBLE/Bluetooth/PeripheralManager+Omnipod.swift

```diff
@@ -89,11 +89,11 @@ extension PeripheralManager {
             var expected: UInt8 = 0
```

Comment:
You can probably remove the force unwrap on line 96 by replacing lines 90 - 94 with:

```
guard let firstPacket = try readData(sequence: expected, timeout: 5) else {
    return nil
}
```